ui/imports/shared/stores/CommunityTokensStore.qml

```diff
@@ -98,14 +102,22 @@ QtObject {
         function onBurnStateChanged(communityId, tokenName, status, url) {
             root.burnStateChanged(communityId, tokenName, status, url)
         }
+    }
 
-        function onBurnFeeUpdated(ethCurrency, fiatCurrency, errorCode) {
-            root.burnFeeUpdated(ethCurrency, fiatCurrency, errorCode)
-        }
+        // Burn:
+    function computeBurnFee(tokenKey, amount, accountAddress, requestId) {
+        console.assert(typeof amount === "string")
```

Comment:
Needed?

Reply:
It's good to have it IMO since the amount needs to be a string.